app/infra/ai/client.py

```diff
@@ -59,7 +59,7 @@ async def get_chat_history(self, session_id: str) -> List[Message]:
 
```

Comment:
[From AI Code Reviewer][bot] Ensure that the renaming of the cache control variable is reflected everywhere in the codebase to avoid runtime errors.